tests/integration/security/authorization_test.go

```diff
@@ -390,87 +388,89 @@ func TestAuthorization_NegativeMatch(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.authorization.negative-match").
 		Run(func(t framework.TestContext) {
-			ns := apps.Namespace1
-			ns2 := apps.Namespace2
+			for _, srcCluster := range t.Clusters() {
+				ns := apps.Namespace1
+				ns2 := apps.Namespace2
+				b := apps.B.Match(echo.Namespace(apps.Namespace1.Name()))
+				c := apps.C.Match(echo.Namespace(apps.Namespace1.Name()))
+				d := apps.D.Match(echo.Namespace(apps.Namespace1.Name()))
+				vm := apps.VM.Match(echo.Namespace(apps.Namespace1.Name()))
+				for _, dst := range [][]echo.Instances{{b, c, d}, {vm, c, d}} {
```

Comment:
Same issue, see comment for TestAuthorization_Deny.

Reply:
Only vm, b, c are selected as targets, and the for loop over target combinations is removed.